reposerver/repository/repository.proto

```diff
@@ -107,6 +108,11 @@ message AppList {
     map<string, string> apps = 1;
 }
 
+// PluginList returns the list of all cmp v2 plugins running as sidecars
+message PluginList {
```

Comment:
We might need to expose more information about plugins for administrators. To avoid painful deprecations can you please change response type to leave room for more information:

```proto
message PluginInfo {
    string name = 1;
}

message PluginList {
    repeated PluginInfo items = 1;
}
```

Reply:
Updated the PR with the suggested change! Thanks